loadtest/loadtest_test.go

```diff
@@ -0,0 +1,92 @@
+package loadtest
+
+import (
+	"context"
+	"testing"
```

Comment:
```suggestion
	"strconv"
	"testing"
```

Reply:
Done and wow I'd totally missed that! TIL #2